### ISSUE TITLE
Windows 10 manifest is missing from youtube-dl.exe [solved by placing external .manifest in same folder]

### ISSUE THREAD

**Issue by eladkarako** [ytdl-org/youtube-dl#29695]
exe without a manifest that isn't stats that the application can run on Windows 10 runs very slowly through a virtualization.  

usually a manifest is embedded in the build process, Windows 7 don't even care about it, but Windows 10 does,  
and it happens to be perfectly fine to just place a text-based file at the same folder, with this content:  

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?> 
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3" xmlns:ms_compatibility="urn:schemas-microsoft-com:compatibility.v1"> 
  <asmv3:trustInfo> 
    <security> 
      <requestedPrivileges> 
        <requestedExecutionLevel level="asInvoker" 
          uiAccess="false"></requestedExecutionLevel> 
      </requestedPrivileges> 
    </security> 
  </asmv3:trustInfo> 
  <ms_compatibility:compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1"> 
    <ms_compatibility:application> 
      <ms_compatibility:supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}"></ms_compatibility:supportedOS> <!-- W10   --> 
      <ms_compatibility:supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}"></ms_compatibility:supportedOS> <!-- W8.1  --> 
      <ms_compatibility:supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}"></ms_compatibility:supportedOS> <!-- W8    --> 
      <ms_compatibility:supportedOS Id="{35138b9a-5d96-4fbd-8e2d-a2440225f93a}"></ms_compatibility:supportedOS> <!-- W7    --> 
      <ms_compatibility:supportedOS Id="{e2011457-1546-43c5-a5fe-008deee3d3f0}"></ms_compatibility:supportedOS> <!-- Vista --> 
    </ms_compatibility:application> 
  </ms_compatibility:compatibility> 
</assembly> 
 ```

a `.manifest` file is attached here for your convenience  
[youtube-dl.exe.manifest.zip](https://github.com/ytdl-org/youtube-dl/files/6910711/youtube-dl.exe.manifest.zip)
place it in the same folder as `youtube-dl.exe`:  

![2021-07-31_075613](https://user-images.githubusercontent.com/415238/127729100-663372d2-f597-4411-bf4b-36d223420d7b.jpg)

**Comment by rautamiekka** [comment]
Having to DL a separate manifest file is bad.

**Comment by eladkarako** [comment]
agreed, you might be can able to integrate it into the `youtube-dl.exe` <del>build-process</del> python-to-exe compilation process,  
might be similar to what one needs to do with manual C compilation (https://github.com/eladkarako/gcc-manifest-icon) but I'm not sure, maybe there is a modification required for the python-to-exe script that is need to be applied (?)  

I wasn't being able to modify the exe afterwards using official ML.EXE tool by Microsoft to modify embedded manifest resource (see https://github.com/eladkarako/manifest) since it seems to corrupt the exe.  

But since Windows 10 is perfectly fine with a side-by-side manifest (<-official name to external manifest) I'm fine with it, it provides a substantial performance boost. Windows 7 users would benefit from it as well and side-by-side manifest works well if the exe isn't having an embedded manifest already (in-which case Windows 7 users should apply a registry patch to prefer the side-by-side manifest over the embedded one), but "I can really feel it" in Windows 10.
